docker-compose.yml

```diff
@@ -0,0 +1,106 @@
+version: '2.4'
+
+services:
+  app: &app
+    build:
+      context: .
+      dockerfile: ./.dockerdev/Dockerfile
+      args:
+        RUBY_VERSION: '2.6.5'
+        PG_MAJOR: '11'
+        NODE_MAJOR: '11'
```

Comment:
let's bump this to 12 since node 11.x is no longer supported

Reply:
Done!